sdk/batch/batch/test/batchClient.spec.ts

```diff
@@ -457,14 +452,14 @@ describe("Batch Service", () => {
     });
 
     it("should add a user to a compute node successfully", async () => {
-      const options = { name: TEST_USER, isAdmin: false, password: "kt#_gahr!@aGERDXA" };
+      const options = { name: TEST_USER, isAdmin: false, password: fakeTestPassword2 };
```

Comment:
Does we read the `fakeTestPassword` from constant file?

Reply:
yes it's read from fakeTestSecrets.ts